### DISCUSSION TITLE
Common practice to create NuGet package for Windows and Linux

### DISCUSSION THREAD

**Discussion by jiten87** [NuGet/Home#11110] [Q&A]
I have to pack OS depended file in NuGet.
suppose abc.dll for Windows and abc.so for Linux.
So what are the options available here.
Should I create 2 packages 1 for windows and others for Linux.
Or We can create a single package.

**Comment by zivkan** [comment]
My suggestion is to find other packages that achieve the same result, and copy how they do it.

I also have a meta-issue, where I've tried to gather links to all the different "how do I create managed wrapper of unmanaged code" issues. You could browse all those issues: https://github.com/NuGet/docs.microsoft.com-nuget/issues/2070
